src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -470,6 +470,10 @@ protected override TDeclarationNode AddMembers<TDeclarationNode>(TDeclarationNod
             {
                 return AddStatementsToMemberDeclaration<TDeclarationNode>(destinationMember, statements, memberDeclaration);
             }
+            else if (destinationMember is LocalFunctionStatementSyntax localFunctionDeclaration)
+            {
+                return AddStatementsToLocalFunction<TDeclarationNode>(destinationMember, statements, localFunctionDeclaration);
+            }
```

Comment:
remove this.  you're not calling AddMembers anymore, right?  you're calling AddStatemnets.

Reply:
Removing this call causes some tests to fail in cases where the destination is a LocalFunctionStatementSyntax.

Reply:
do you mean 'extract method' tests?  or some other feature?  if the latter, i feel like it's an issue with teh feature.  if the destination if a local-function, it should be using AddStatements as well as the only thing it should be able to add to a local function is another local function.